transport/server.cc

```diff
@@ -601,12 +601,10 @@ future<> cql_server::connection::process_request() {
         auto stream = f.stream;
         auto mem_estimate = f.length * 2 + 8000; // Allow for extra copies and bookkeeping
         if (mem_estimate > _server._max_request_size) {
```

Comment:
This is pre-existing code, but I think it's very strange that instead of limiting the request size, we limit some silly function len*2+8000. Wouldn't it be more logical for the user to be told that his request is longer than the limit of 100,000 bytes - instead of being told that 2*len+8000 is longer than 208,000 bytes?

Reply:
Right, it never caught my eye, but comparing to `f.length` sounds more natural here. It's a good candidate for a follow-up patch, but it may on the other hand ruin existing configurations for some users, since their clusters would suddenly start accepting requests twice as large as the maximum before the patch.